lib/apivore.rb

```diff
@@ -2,6 +2,7 @@
 require 'apivore/rspec_helpers'
 require 'apivore/swagger_checker'
 require 'apivore/swagger'
+require 'rspec/rails'
```

Comment:
I've tried with just `require 'rspec'` and that prevents the erroring. I was trying to be more specific rather than general.

I thought it was included as a runtime dependency already, but it only `rspec-expectations` and `rspec-mocks`. `rspec` and `rspec-rails` are only development dependencies at the moment. I'll add what we go with here as a runtime dep.

What is better, `rspec`, `rspec-rails`, or is there something else that will allow the `RSpec.configure` at https://github.com/westfieldlabs/apivore/blob/master/lib/apivore.rb#L6 to work in all situations?